src/koaApi.js

```diff
@@ -25,15 +27,41 @@ import * as tasks from './api/tasks'
 import * as transactions from './api/transactions'
 import * as users from './api/users'
 import * as visualizers from './api/visualizers'
+import passport from './passport'
+import MongooseStore from './middleware/sessionStore'
 import {config} from './config'
 
 export function setupApp(done) {
-  // Create an instance of the koa-server and add a body-parser
+  // Create an instance of the koa-server
   const app = new Koa()
-  app.use(cors({allowMethods: 'GET,HEAD,PUT,POST,DELETE'}))
+
+  // Add cors options
+  app.use(cors({allowMethods: 'GET,HEAD,PUT,POST,DELETE', credentials: true}))
+
+  // Configure Sessions Middlewares
+  app.keys = [ 'r8q,+&1LM3)CD*zAGpx1xm{NeQhc;#' ]
```

Comment:
Is this the session salt ? Could we make this a configuration ?

Reply:
cool done :)